Neptune/Neptune.csproj

```diff
@@ -4,4 +4,10 @@
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\newtonsoft.json\11.0.2\lib\netstandard2.0\Newtonsoft.Json.dll</HintPath>
```

Comment:
Wait what is this for? I've never seen this being used

Reply:
Newtonsoft is a popular json ser/deser library. Has filter, flatMap and map operators for algebraic json types, and therefore can be used very cleanly with LINQ. see: https://www.newtonsoft.com/json/help/html/LINQtoJSON.htm

Reply:
It's really useful. Can confirm.

Reply:
oh yes i agree. I have used newtonsoft before. I just have never seen a hint path.